packages/swingset/CLAUDE.md

```
@@ -0,0 +1,35 @@
+# CLAUDE.md
+
+This file provides guidance to Coding Agents when working with code in this repository.
+
+`@clerk/swingset` is a private (unpublished) component explorer — a Storybook-like app — for the **Mosaic** design system that lives in `@clerk/ui`. It is a Next.js App Router app that renders Mosaic components interactively with live knobs and design-token overrides.
+
+## Commands
+
+Run from the monorepo root or scope with `--filter @clerk/swingset`:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)